docker/sandbox/flyte-entrypoint-default.sh

```diff
@@ -18,7 +18,7 @@ echo "Done."
 # Deploy flyte
 echo "Deploying Flyte..."
 helm dep update /flyteorg/share/flyte/
-helm install -n flyte -f /flyteorg/share/flyte/values-sandbox.yaml --create-namespace flyte /flyteorg/share/flyte --kubeconfig /etc/rancher/k3s/k3s.yaml
+helm upgrade -n flyte -f /flyteorg/share/flyte/values-sandbox.yaml --create-namespace flyte /flyteorg/share/flyte --kubeconfig /etc/rancher/k3s/k3s.yaml --install
```

Comment:
Can you add in the description how does this change fix the issue?

Reply:
done